go/engine/per_user_key_background.go

```diff
@@ -0,0 +1,193 @@
+// Copyright 2017 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+// PerUserKeyBackground runs PerUserKeyUpgrade in the background once in a while.
+// It brings users without per-user-keys up to having them.
+// Note that this engine is long-lived and potentially has to deal with being
+// logged out and logged in as a different user, etc.
```

Comment:
I have no idea what the answer to that is, so maybe just something we should make sure to test by hand.